### PR TITLE
Move styling from RadioSchedule to Home/IDX

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Part of #7215

**Overall change:**
Moving margin styling out of the Radio Schedule and into FrontPage/IdxPage. This means the RadioSchedule component can now be reused by the Live Radio and On Demand Radio pages without them inheriting unwanted styling.

**Code changes:**

- Moved media queries from RadioSchedule to FrontPage/IdxPage
- Pass down className to Canonical
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]

**Review by karinathomasbbc — APPROVED** [review]
👍

**Review by tochwill — APPROVED** [review]

**Comment by PriyaKR** [comment]
Regression on frontpage looks good.

File: src/app/pages/FrontPage/index.jsx [hunk 1]
```diff
@@ -32,6 +43,20 @@ const FrontPageMostReadSection = styled(MostReadSection)`
   }
 `;
 
+const StyledRadioScheduleContainer = styled(RadioScheduleContainer)`
```
Comment (DenisHdz) [inline src/app/pages/FrontPage/index.jsx:46]: Sorry for being a bit late to the review, but would it be possible to export these styles from the Radio Schedule folder? So we can import them in both, FrontPage and IDX pages? 
Also, we have been trying to keep the page containers without styling definitions to reduce their size.

Comment (rebeccamcginn) [reply src/app/pages/FrontPage/index.jsx:46]: Hi Denis, do you have something along these lines in mind? https://github.com/bbc/simorgh/pull/7260

Comment (DenisHdz) [reply src/app/pages/FrontPage/index.jsx:46]: That's exactly what I meant 👌🏻

**Pull request merged**